pom.xml

```diff
@@ -309,8 +309,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
is this change needed ? It's failing on my setup. This will use JDK 1.8 but my environment is setup for 1.7